frontend/src/components/landing/Reviews.module.scss

```
@@ -0,0 +1,294 @@
+    justify-content: center;
+
+    @media screen and #{$mq-lg} {
+      gap: $layout-md;
+      flex-direction: row;
+    }
+  }
+}
+
+.left-container {
+  display: flex;
+  flex-direction: row;
+  gap: $spacing-xl;
+}
+
+.right-container {
+  display: flex;
+  flex-direction: column;
+  gap: $spacing-2xl;
+}
+.logo-container {
+  display: flex;
+  font-family: $font-stack-base;
+  color: $color-white;
+  width: $layout-2xl;
+  flex-direction: column;
+  align-items: center;
+  border-radius: $border-radius-lg;
+  padding: $spacing-md 0;
+  background-color: $color-purple-30;
+
+  .logo {
+    width: $layout-xl;
+    padding: $spacing-sm;
+  }
+
+  .logo-title {
+    @include text-body-md;
+    font-weight: 400;
+  }
+
+  .logo-text {
+    @include text-title-xs;
+    font-weight: 600;
+    text-transform: uppercase;
+  }
+}
+
+.rating-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  gap: $spacing-xs;
+
+  .rating {
+    .title {
+      @include text-title-2xl;
+      font-family: $font-stack-firefox;
+      font-weight: 600;
+    }
+
+    .text {
+      @include text-body-sm;
+      color: $color-grey-40;
+      font-family: $font-stack-base;
+      font-weight: 500;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.star {
+  color: $color-yellow-50;
+}
+
+.empty-star {
+  color: $color-white;
+  stroke: $color-grey-20;
+  stroke-width: 2px;
+}
+
+.chevron {
+  appearance: none;
+  border: none;
+  padding: 0;
+  background-color: $color-white;
+  color: $color-grey-30;
+  width: $layout-sm;
+  cursor: pointer;
+}
+
+.quotation-icon {
+  display: flex;
+  position: absolute;
+
+  @media screen and #{$mq-md} {
+    left: 0;
+  }
+
+  svg {
+    color: $color-purple-60;
+    height: $layout-sm;
+  }
+}
+
+.reviews-container {
+  display: flex;
+  overflow-x: hidden;
+  flex-direction: row;
+  gap: $spacing-lg;
+  // magic number: set to avoid shifting - no character limit on reviews yet.
+  min-height: calc($layout-2xl * 2);
+  position: relative;
+
+  @media screen and #{$mq-xl} {
+    // magic number: set to avoid shifting - no character limit on reviews yet.
+    min-height: 210px;
+  }
+
+  .review-container {
+    display: flex;
+    align-items: center;
+
+    @media screen and #{$mq-xl} {
+      align-items: flex-start;
+    }
+  }
+
+  .review {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    position: relative; // allows us to animate - scroll
+    width: 100%; // allows us to animate - scroll
+    padding: 0 $spacing-sm;
+
+    // queries set to these static widths to prevent shifting
+    @media screen and #{$mq-sm} {
+      width: $content-xs;
+    }
+    @media screen and #{$mq-md} {
+      width: $content-sm;
+    }
+    @media screen and #{$mq-xl} {
+      width: $content-md;
+    }
+
+    .text {
+      width: 100%;
+
+      ul {
+        padding-left: calc($spacing-sm + $spacing-xs);
```

Would a `$spacing-md` not suffice here, to ensure it's consistent with spacings elsewhere? The list icon circles appear to align better with the star ratings that way, to my eye.

```suggestion
        padding-left: $spacing-md;
```